src/components/Identicon/StatusIcon.tsx

```diff
@@ -70,7 +67,8 @@ const useIcon = (connectionType: ConnectionType) => {
 }
 
 export default function StatusIcon({ connectionType, size }: { connectionType: ConnectionType; size?: number }) {
-  const hasSocks = useHasSocks()
+  //const hasSocks = useHasSocks()
```

Comment:
make sure to uncomment this if you're done and want to merge this

Reply:
Oops, sorry! Fixed in the latest commit